code/modules/mining/ores_coins.dm

```diff
@@ -23,9 +23,13 @@
 	..()
 
 /obj/item/weapon/ore/Crossed(atom/movable/AM)
+	var/show_message = TRUE
+	for(var/obj/item/weapon/ore/O in loc)

```

Comment:
That's a good point. I'm not sure if there's a better way to do it, but I'll see if I can think of something.